src/coreclr/src/jit/optcse.cpp

```diff
@@ -2299,6 +2405,8 @@ class CSE_Heuristic
     // It will also put cse0 into SSA if there is just one def.
     void PerformCSE(CSE_Candidate* successfulCandidate)
     {
+        const char* grabTempMessage = "CSE - unknown";
+
```

Comment:
This and related code should probably be under `DEBUG`

Reply:
Resolved